### ISSUE TITLE
Pop-up menu appears without me intentionally triggering it.

### ISSUE THREAD

**Issue by Bortseb** [Pitmairen/selection-search#29]
I am going to link the other issue to this one, because I may have confused the other issue I think... and in reality, this is a new issue that I have recently been having.

Selection search has become an integral component of how I surf the web. However, recently the pop-up menu has been appearing, when I don't want it to, simply when I left-click on links. I usually use middle mouse click to activate the pop-up menu (However, I just looked at the options for the extension... and now I am totally confused with how I ever customized this. The only relevant settings I see is that I have selected "Multiple activators" with mouse click and keyboard+mouse selected... I'm not sure how I chose middle mouse click in particular anymore)

I really don't want to disable this extension. I still use it every day... but its getting annoying to have to realod pages to click on links (Which is the quickest workaround that I have figured out)

**Comment by Pitmairen** [comment]
Yes, the configuration of the activators is a little bit confusing. When multiple activators is selected it uses the settings that is set for each of the activators when they are selected in the drop down menu. So to configure the "Mouse Click" activator you have to select this activator from the drop down menu and do the configuration and then switch back to multiple activators,

But I haven't had this issue, but i'm not using multiple activators. Does it still happen if you don't use multiple activators? Are there any particular websites where this usually happens or does it happen on all sites?

**Comment by Bortseb** [comment]
No pattern based on domain... I just turn off multiple activators... I actually forget why it was on, and wasn't using my keyboard+mouse anymore... I will report back if the problem persists,
